lib/gameplay.rb

```diff
@@ -1,4 +1,6 @@
 class Gameplay
+  attr_reader :player_board,
```

Comment:
I don't think we need this attr_reader beacuse we don't need to call upon these board objects outside of the class (at least not yet).